packages/metacoin/package.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Should `Metacoin` be in the monorepo?! Could see both sides of the argument. What I'm worried about is people forking the monorepo and building their project inside it 🔫  But maybe that's a stupid worry...

Reply:
I'm ... not sure. I think it might make sense to move it out later. It's much easier to develop like that. Let's keep it for now and move out before we push it hard to the public.